### ISSUE TITLE
[FEATURE] UNSAT Verification

### ISSUE THREAD

**Issue by hadarshavit** [ADA-research/Sparkle#116]
**Is your feature request related to a problem? Please describe.**
N / A
**Describe the solution you'd like**
Currently, sparkle has functionality to verify SAT results. However, in recent years, UNSAT verification is mandatory in SAT Competitions. Add an option in sparkle to verify UNSAT proofs.

**Describe alternatives you've considered**
Not verifying UNSAT proofs. Some users might be interested in UNSAT proofs as well as they are increasingly becoming "part of SAT solving"

**Additional context**
https://satcompetition.github.io/2024/output.html (Certificates of Unsatisfiablity section)

**Comment by thijssnelleman** [comment]
Agreed, currently present in Sparkle settings file but this makes no sense. Should be present on Solver level, s.t. the verifier is always given when a solver is loaded. Would also allow for different verifiers in different solvers if needed. Will try to add in tomorrow

**Comment by thijssnelleman** [comment]
@hadarshavit this is fixed now
